local_setup.md

```diff
@@ -0,0 +1,193 @@
+# Local Setup
+
+For developing you want to set up the renuo-cms and the renuo-upload locally. They must be set up once and can then be used by every application that uses one or both of those services.
+
+## Step by Step Manual
+
+### Install renuo-cms-api
+
+Clone git repository and run bin/setup:
+
+```
+git clone git@github.com:renuo/renuo-cms-api.git
+cd renuo-cms-api
+bin/setup
+```
+
+If you already cloned the project, you can just pull the newest version and migrate the database.
+
+### Install renuo-upload signing
```

Comment:
this whole part should be described in the renuo upload already? if not: link to the renuo upload doc, or link the renuo upload doc to here. i would create a separate document for setting up the renuo upload.